sktime/classification/shapelet_based/_rocket_classifier.py

```diff
@@ -109,23 +122,16 @@ def fit(self, X, y):
             self.class_dictionary[classVal] = index
 
         if self.ensemble:
-            for i in range(self.ensemble_size):
-                rocket_pipeline = make_pipeline(
-                    Rocket(
-                        num_kernels=self.num_kernels, random_state=self.random_state
-                    ),
-                    RidgeClassifierCV(alphas=np.logspace(-3, 3, 10), normalize=True),
-                )
-                rocket_pipeline.fit(X, y)
-                self.classifiers.append(rocket_pipeline)
-                self.weights.append(rocket_pipeline.steps[1][1].best_score_)
-                self.weight_sum = self.weight_sum + self.weights[i]
-        else:
-            rocket_pipeline = make_pipeline(
-                Rocket(num_kernels=self.num_kernels, random_state=self.random_state),
-                RidgeClassifierCV(alphas=np.logspace(-3, 3, 10), normalize=True),
+            self.classifiers = Parallel(n_jobs=self.n_jobs)(
+                delayed(_fit_pipeline)(X, y, self.num_kernels, self.random_state)
```

Comment:
Here we would pass the cloned instance
```suggestion
                delayed(_fit_estimator)(clone(estimator), X, y)
```